Automattic/wp-calypso
# Delete Site: Redirect to /sites instead of /read after deleting the site

Fixes https://github.com/Automattic/dotcom-forge/issues/5101

## Proposed Changes

This pr aims to fix the "default redirect" when we delete a site. Until now, when the user deleted a site and there wasn't one left, it got redirected to `/read` page. We change that now to be the Sites Dashboard ( `/sites` )

## Testing Instructions

1. Open incognito mode, and create a new user.
2. Purchase a new free site.
3. Navigate to `/settings/delete-site/<your site`
4. Press the `Delete Site` button
5. Ensure that you get redirect at `/sites` page and you see the following image

![2024-01-12T12:33:40,423523901+02:00](https://github.com/Automattic/wp-calypso/assets/497103/947ee659-809a-47e5-b8de-d94462f290d6)


## Pre-merge Checklist
